sdk/iot/core/tests/cmocka/main.c

```diff
@@ -57,6 +57,16 @@ void test_az_iot_hub_client_telemetry_publish_topic_get_with_options_user_agent_
  * IoT Hub Client Unit Tests
  */
 void test_az_iot_hub_client_get_default_options_succeed(void** state);
+void test_az_iot_hub_client_init_succeed(void** state);
+void test_az_iot_hub_client_init_custom_options_succeed(void** state);
+void test_az_iot_hub_client_user_name_get_succeed(void** state);
+void test_az_iot_hub_client_user_name_get_small_buffer_fail(void** state);
+void test_az_iot_hub_client_user_name_get_user_options_succeed(void** state);
+void test_az_iot_hub_client_user_name_get_user_options_small_buffer_fail(void** state);
+void test_az_iot_hub_client_id_get_succeed(void** state);
+void test_az_iot_hub_client_id_get_small_buffer_fail(void** state);
+void test_az_iot_hub_client_id_module_get_succeed(void** state);
+void test_az_iot_hub_client_id_module_get_small_buffer_fail(void** state);
```

Comment:
how about sorting them case on name of the function?

Reply:
I can bring this up in the meeting today but the policy on unit testing framework (grouping, where functions are declared etc) will be discussed today.